docs/RUP_Architecture.md

```diff
@@ -0,0 +1,131 @@
+
+#### Scope
+The scope of this Software Architecture document is to show the architecture of the FiBo project. We illustrate the use cases and the overall structure.
+
+#### Definitions, Acronyms and Abbreviations
+
+| Abbreviation | Description |
+| --- | --- |
+| API | Application Programming Interface |
+| SRS | Software Requirement Specification |
+| n/a | not applicable |
+| UC | use case |
+| CRUD | create-read-update-delete |
+
+#### References
+
+| Name and link | Date | Publishing organization |
+| --- | --- | --- |
+| [Blog](https://fibo952390745.wordpress.com/) | ongoing | FiBo project team |
+| [Git repository](https://github.com/Cebox82/FiBo) | ongoing | FiBo project team |
+| [SRS](https://github.com/Cebox82/FiBo/blob/master/docs/SRS.md) | 16/11/2022 | FiBo project team |
+| [UC 4 - Manual adding of data](https://github.com/Cebox82/FiBo/blob/master/docs/use_cases/UC_4_manual_adding_of_data/UC_4_manual_adding_of_data.md) | 21/11/2022 | FiBo project team |
+
+#### Overview
+
+This document contains the Architectural Representation, Goals and Constraints as well as the Logical, Deployment, Implementation and Data Views.
+
+### Architectural Representation
+
+The backend (Django based) and the frontend are both developed separted from each other and only communicate over a REST API. That said, it is hard to follow one of the known patterns (MVC, MVP, MVVM) for the whole project, as the frontend ecosystem (an Android app) can only be so much used to follow one of some known pattterns.
+
+Please also see this [introduction to the technologies](https://fibo952390745.wordpress.com/2022/10/18/week-2-roles-and-technologies/).
+
+![Overview of the Architecture](architectural_UML_Diagram.drawio.svg "Architecture")
+
+### Architectural Goals and Constraints
+
+[This section describes the software requirements and objectives that have some significant impact on the architecture, for example, safety, security, privacy, use of an off-the-shelf product, portability, distribution, and reuse. It also captures the special constraints that may apply: design and implementation strategy, development tools, team structure, schedule, legacy code, and so on.]
+
+### Use-Case View
+
+[This section lists use cases or scenarios from the use-case model if they represent some significant, central functionality of the final system, or if they have a large architectural coverage - they exercise many architectural elements, or if they stress or illustrate a specific, delicate point of the architecture.]
+
+#### Use-Case Realizations
+
+[This section illustrates how the software actually works by giving a few selected use-case (or scenario) realizations, and explains how the various design model elements contribute to their functionality.]
+
```

Comment:
Should we also include the Mockups here?

Reply:
if it is not necesary then rather not. After all, we have them in the repo